src/com/amplitude/api/DatabaseHelper.java

```diff
@@ -54,10 +61,38 @@ public void onCreate(SQLiteDatabase db) {
 
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        db.execSQL("DROP TABLE IF EXISTS " + AMPLITUDE_STORE_TABLE_NAME);
```

Comment:
read about SQLiteOpenHelper onUpgrade. You'll need to do version by version upgrading, to preserve old data (we've never really had to use this method before)